2-mandatory/3-shopping-list.js

```diff
@@ -19,7 +19,13 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
```

Comment:
I like your use of the `filter` array method here 😄 
One small point: Do you need to assign the result of the `filter` method back to `recipe["items"]`? 🤔